funnel/views/participant.py

```diff
@@ -115,7 +115,7 @@ def participant_edit(profile, space, participant):
     return render_form(form=form, title=_(u"Edit Participant"), submit=_(u"Save changes"))
 
 
-@app.route('/<space>/participant', subdomain='<profile>')
+@app.route('/<space>/participant', methods=['POST'], subdomain='<profile>')
```

Comment:
This will break current Android apps. Keep both GET and POST and leave a comment that GET is deprecated. We'll need to set a reminder to remove it in a few months.